Spotlight: BeforeSunset AI & Product Hunt launches (#6608)
PostHog/posthog.com: contents/spotlight/startup-before-sunset-ai.md

---

> title: "Startup Spotlight: How BeforeSunset AI mastered Product Hunt launches"

Reviewer: ```suggestion
title: "How BeforeSunset AI achieved 'Product of the Month' on Product Hunt"
```

Feel free to disregard, but if we have the custom image we can drop the "Startup Spotlight" kicker and go with something like this. Adding they were product of the month validates what "mastered" really means. That's a massive, eye-opening achievement, right?

“The first version of BeforeSunset was just an internal time tracking tool I’d built,” explains Elif. “But when we launched as a separate venture we saw that no other productivity tool asks you the questions needed to plan a day properly. We realized that AI could help if it had the right information.”

After rebuilding the product around this AI-powered vision, the team wanted to relaunch in earnest — and in June 2023 debuted on Product Hunt, going on to win [Product of the Day, Week, and Month](https://www.producthunt.com/products/beforesunset). Here’s how:

![BeforeSunset AI Launch on Product Hunt](../images/blog/before-sunset-ai.png)

### 1. Create a product launch plan in advance

Unsurprisingly for a productivity startup, it all began with a plan.

“We had a clear plan in our minds for the product,” says Elif. “We wanted to complete the core features, such as to-do listing and time blocking, and we also wanted to build in AI features. In total it took about three months before we were ready to launch to users.”

> This development time afforded the growth team time to plan, while also giving them the benefit of knowing how the product would come together. Work was hard — to the point that Elif worked several 40-hour sprints without sleeping — but came together smoothly. 

Author: ```suggestion
This development time afforded the growth team time to plan, while also giving them the benefit of knowing how the product would come together. Work was hard and required some long nights, but came together smoothly. 
```

“On the marketing side our efforts started six months before the launch,” says Growth Lead Zekiye Nur Kesici. “In addition to on-going activities we began a waitlist for the new version and began growing it. We tried many different channels, including Reddit, Indie Hackers, Hacker News — and we spent a month on competitor analysis to find new channels too.”

> **Putting it into action:** For new product launches take steps to align your product and marketing team as early as possible, and make sure they stay in sync as development continues. Product managers should keep marketing teams informed of key features and delays, so marketing can plan how to position the launch and manage deliverables. Product marketers, unsurprisingly, can help bridge the gap. 

### 2. Prioritize community and creativity

Focusing on community engagement is an important part of any launch plan, according to Zekiye — and the earlier you start, the more successful you’ll be. 

> “We chose to focus on the Product Hunt community and we knew that would require us to be active on the platform,” she explains. “That means finding ways to add value; ways you can support them whenever you’re in touch so that, in turn, you can benefit from the community.”

Author: ```suggestion
“We chose to focus on the Product Hunt community and we knew that would require us to be active on the platform,” she explains. “That means finding ways to add value; ways you can support them whenever you’re in touch so that, in turn, you can benefit from the community.”

For BeforeSunset, this included engaging with the community regularly and offering expertise where possible. 
```

"We're always working on better defining our target audience," says Zekiye. "Nailing the persona is especially hard, because we often see users interacting with the product in ways we don't expect."

One example of this came from BeforeSunset's 'Plan my day' tool, where users are intended to add tasks and let the AI schedule time for them. [Session replays](/session-replay) showed users were instead adding their own timeblocks in the description, which helped the team understand how busy their users were and what tasks they spent their time on.

> **Putting it into action:** Document your target audiences and personas somewhere that everyone can access, then use this to identify the communities where these users may congregate. Use the persona as a benchmark for the quality of your marketing assets, and be sure to engage with communities well in advance of a launch to build goodwill. 

### 3. Create an outreach strategy

With Product Hunt, you only have a short amount of time in order to get traction and galvanize the community. Take too long and you’ll fall off the daily page, and into the archives. So, you need a strategy to continue currying favor and votes throughout the day.

> “We created a spreadsheet of our users and ran some one-on-one outreach to them,” says Zekiye Nur Kesici. “We even had a competition in the marketing team, offering a prize to whoever could reach out to the most people via LinkedIn. We also ran a webinar, published a blog post, used social media, and leveraged our waitlist to collect votes.”

Reviewer: Random thought... could they provide a screenshot / example of one of their outreach messages. Would really lift this if we could say here's how to write a good message asking for a vote.

"That initial launch was still a milestone for us," says Elif. "It enabled us to get product feedback and led us to discover the real need in the market, which nobody was serving...It’s always important to bear in mind that success is rarely straightforward and that setbacks are a normal part of being an entrepreneur."

In BeforeSunset's case, the team responded to the first launch by speaking to users and spending a month on intense competitor analysis. This led Elif to realize that AI was the missing ingredient and that, by asking users the right questions, AI could plan time much more effectively. Without this insight, BeforeSunset may never have had it's later breakthrough. 

“You must be patient and persistent; have faith in your ability to find answers or make a new path," advises Elif. "You have to be strong all the time, and know you are not alone. A good team makes everything easier, greater. Product is important, but humans are everything.”

> > **Putting it into action:** Failure is the best teacher, so if a product announcement doesn't go as planned then make sure to run a post-mortem process and think of how it can be improved. It can also help to use [surveys](/docs/surveys/manuals) to reach out to existing users and ask them for their advice directly!

Reviewer: Shall we link to other spotlight guides here?
